libs/user-identity-service/src/test/java/uk/gov/di/ipv/core/library/service/UserIdentityServiceTest.java

```diff
@@ -1791,6 +1795,29 @@ void findIdentityThrowsHttpResponseExceptionWithErrorBodyWhenNoNamePresent() {
                 () -> userIdentityService.findIdentityClaim(vcStoreItems, false));
     }
 
+    @Test
+    void findIdentityReturnsIdentityClaimForOperationalVC()
+            throws HttpResponseExceptionWithErrorBody, CredentialParseException {
+        VcStoreItem vcStoreItem =
+                TestFixtures.createVcStoreItem(USER_ID_1, HMRC_MIGRATION_CRI, VC_HMRC_MIGRATION);
+        List<VcStoreItem> vcStoreItems = new ArrayList<>();
+        vcStoreItems.add(vcStoreItem);
+        Optional<IdentityClaim> result = userIdentityService.findIdentityClaim(vcStoreItems);
+        assertFalse(result.isEmpty());
+    }
+
+    @Test
+    void findIdentityDoesNotReturnsIdentityClaimForOperationalVcWithNoEvidence()
```

Comment:
Name of the test doesn't match the assertion - I think it should be `findIdentityReturnsIdentityClaimForOperationalVcWithNoEvidence` (we do get an identity claim!)

Reply:
Aah forgot to change...was created while testing the bug. Done!